doc/Onboarding/Markup/Overview.md

```diff
@@ -0,0 +1,222 @@
+                                .HorizontalAlignment(HorizontalAlignment.Center)
+                                .HorizontalTextAlignment(Microsoft.UI.Xaml.TextAlignment.Center)
+                                .Text(() => vm.CounterValue, txt => $"Counter: {txt}"),
+                            new Button()
+                                .Margin(12)
+                                .HorizontalAlignment(HorizontalAlignment.Center)
+                                .Command(() => vm.IncrementCommand)
+                                .Content("Click me to increment Counter by Step Size")
+                        )
+                    )
+            );
+    }
+}
+```
+
+Looking at this full example, it's easy to see how C# Markup provides a succinct, self-documenting syntax for defining the layout of an application. However, setting the `Margin` and `HorizontalAlignment` for each control is a bit repetitive. The above code could be simplified by creating a helper extension method that sets the `Margin` and `HorizontalAlignment` for each control, as shown in the following example.
+
+```csharp
+public static class MainPageHelpers
+{
+    public static T CenterAndSpace<T>(this T element) where T : FrameworkElement
+        => element
+            .HorizontalAlignment(HorizontalAlignment.Center)
+            .Margin(12);
+    
+}
+```
+Using the `CenterAndSpace` extension method, the code can be simplified to the following:
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    public MainPage()
+    {
+        this
+            .DataContext(
+                new MainViewModel(),
+                (page, vm) => page
+                    .Background(Theme.Brushes.Background.Default)
+                    .Content(new StackPanel()
+                        .VerticalAlignment(VerticalAlignment.Center)
+                        .Children(
+                            new Image()
+                                .Width(150)
+                                .Height(150)
+                                .CenterAndSpace()
+                                .Source("ms-appx:///Counter/Assets/logo.png"),
+                            new TextBox()
+                                .CenterAndSpace()
+                                .PlaceholderText("Step Size")
+                                .Text(x => x.Bind(() => vm.StepSize).TwoWay()),
+                            new TextBlock()
+                                .CenterAndSpace()
+                                .HorizontalTextAlignment(Microsoft.UI.Xaml.TextAlignment.Center)
+                                .Text(() => vm.CounterValue, txt => $"Counter: {txt}"),
+                            new Button()
+                                .CenterAndSpace()
+                                .Command(() => vm.IncrementCommand)
+                                .Content("Click me to increment Counter by Step Size")
+                        )
+                    )
+            );
+    }
+}
+```
```

Comment:
I get what you're doing but this isn't the right time or place. Please remove this.